Day7/index.js

```diff
@@ -0,0 +1,72 @@
+$( () => {
+
+  let score = 0;
+  let newTurn = true;
+  let lastClick;
+  let emoji = [ '💩' , '🐝' , '🍙' , '🐒' , '🌚' ];
+
+  const shuffle = ( myArray ) => {
+    for( let i = myArray.length-1 ; i >= 0 ; i-- ) {
+      let randomIndex = Math.floor( Math.random()*(i+1) );
+      let temp = myArray[randomIndex];
+      myArray[randomIndex] = myArray[i];
+      myArray[i] = temp;
+    }
+    return myArray;
+  };
+
+  const buttonOpen = ( click ) => {
+    $( '#'+click ).attr( 'class' , 'btn btn-link');
```

Comment:
Use `$( '#'+click ).attr( 'class' , 'btn');` is enough to set background to white.